proto/desmos/posts/v1beta1/msgs.proto

```diff
@@ -48,9 +48,9 @@ message MsgCreatePost {
 
   string message = 2 [ (gogoproto.moretags) = "yaml:\"message\"" ];
 
-  bool allows_comments = 3 [
-    (gogoproto.jsontag) = "allows_comments",
-    (gogoproto.moretags) = "yaml:\"allows_comments\""
+  CommentsState comments_state = 3 [
```

Comment:
Can we rename this to simply be `comments` since it will be either `comments: allowed` or `comments: blocked`?

Reply:
This is still named `comments_state`